Enforce the usage of --kubeconfig flag in Solutions script

Ebaneck opened the pull request:
**Component**:

<!-- E.g. 'salt', 'containers', 'kubernetes', 'build', 'tests'... -->
'kubernetes'

**Context**: 

While using Solutions.sh script to provision new solutions, users fall into the following situations:
The connection to the server localhost:8080 was refused - did you specify the right host or port?

This is likely because we didn't specify the `--kubeconfig` flag while executing kubectl commands

**Summary**:

Most Devs will have an environment with the exported KUBECONFIG env but this is not the case in the CI. And since we need to test the example-solution in the CI, we require to explicitly set the `--kubeconfig` flag.

**Acceptance criteria**: 

- Working Solutions.sh script

---

<!-- Declare one or more issues to close once this PR gets merged -->

Closes: #ISSUE_NUMBER

<!-- If you want to refer to an issue while not closing it, use:

See: #ISSUE_NUMBER

-->

bert-e commented:
# Hello ebaneck,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/add-kubeconfig-to-solutions-script` into
`development/2.4`
* [w/2.5/bugfix/add-kubeconfig-to-solutions-script](https://github.com/scality/metalk8s/compare/development/2.5...w/2.5/bugfix/add-kubeconfig-to-solutions-script) will be merged into `development/2.5` 
* [w/2.6/bugfix/add-kubeconfig-to-solutions-script](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/bugfix/add-kubeconfig-to-solutions-script) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @tcarmet

* @jbertran

* @carlito-scality

* @slaperche-scality

* @nicolast

* @teddyandrieux

* @monpote

* @sayf-eddine-scality

* @nootal

* @alexandre-allard-scality

* @gdemonet

* @lucieleonard

* @thomasdanan

* @cmonfort

NicolasT reviewed (changes requested):
Can't you just `export KUBECONFIG` somewhere in the beginning of this script and be done with it? :thinking:

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @tcarmet

* @jbertran

* @carlito-scality

* @slaperche-scality

* @nicolast

* @teddyandrieux

* @monpote

* @sayf-eddine-scality

* @nootal

* @alexandre-allard-scality

* @gdemonet

* @lucieleonard

* @thomasdanan

* @cmonfort





The following reviewers are expecting changes from the author, or must review again:

* @nicolast

bert-e commented:
# History mismatch



Merge commit #c5e5c98ceb2d663970124f95d81be20b92b50e54 on the integration branch
`w/2.5/bugfix/add-kubeconfig-to-solutions-script` is merging a branch which is neither the current
branch `bugfix/add-kubeconfig-to-solutions-script` nor the development branch
`development/2.5`.

It is likely due to a rebase of the branch `bugfix/add-kubeconfig-to-solutions-script` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

Ebaneck committed:
script: make use of kubeconfig in solutions script

While using the solutions script, users fall into the following issues
The connection to the server localhost:8080 was refused - did you specify the right host or port?`

This is as a result of no kubeconfig flag specified while perform the kubectl commands.
This addition is especially useful since we need to automate example-solutions in the CI

Ebaneck commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/add-kubeconfig-to-solutions-script` into
`development/2.4`
* [w/2.5/bugfix/add-kubeconfig-to-solutions-script](https://github.com/scality/metalk8s/compare/development/2.5...w/2.5/bugfix/add-kubeconfig-to-solutions-script) will be merged into `development/2.5` 
* [w/2.6/bugfix/add-kubeconfig-to-solutions-script](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/bugfix/add-kubeconfig-to-solutions-script) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @tcarmet

* @jbertran

* @carlito-scality

* @slaperche-scality

* @nicolast

* @teddyandrieux

* @monpote

* @sayf-eddine-scality

* @nootal

* @alexandre-allard-scality

* @gdemonet

* @lucieleonard

* @thomasdanan

* @cmonfort





The following reviewers are expecting changes from the author, or must review again:

* @nicolast

Ebaneck commented:
/approve

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @tcarmet

* @jbertran

* @carlito-scality

* @slaperche-scality

* @nicolast

* @teddyandrieux

* @monpote

* @sayf-eddine-scality

* @nootal

* @alexandre-allard-scality

* @gdemonet

* @lucieleonard

* @thomasdanan

* @cmonfort





The following reviewers are expecting changes from the author, or must review again:

* @nicolast







*The following options are set:* **approve**

NicolasT commented on scripts/solutions.sh:
```diff
@@ -20,6 +20,8 @@ SOLUTION=''
 VERBOSE=${VERBOSE:-0}
 VERSION=''
 
+export KUBECONFIG="$KUBECONFIG"
```
`export KUBECONFIG` should be enough (and more portable).

Ebaneck replied:
Ok, I see

Done!

bert-e commented:
# History mismatch



Merge commit #e35b7c7319c7fadb1d3450d8a4ee8092a7dd32c8 on the integration branch
`w/2.5/bugfix/add-kubeconfig-to-solutions-script` is merging a branch which is neither the current
branch `bugfix/add-kubeconfig-to-solutions-script` nor the development branch
`development/2.5`.

It is likely due to a rebase of the branch `bugfix/add-kubeconfig-to-solutions-script` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

Ebaneck commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.






*The following options are set:* **approve**

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/add-kubeconfig-to-solutions-script` into
`development/2.4`
* [w/2.5/bugfix/add-kubeconfig-to-solutions-script](https://github.com/scality/metalk8s/compare/development/2.5...w/2.5/bugfix/add-kubeconfig-to-solutions-script) will be merged into `development/2.5` 
* [w/2.6/bugfix/add-kubeconfig-to-solutions-script](https://github.com/scality/metalk8s/compare/development/2.6...w/2.6/bugfix/add-kubeconfig-to-solutions-script) will be merged into `development/2.6` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```







*The following options are set:* **approve**

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @tcarmet

* @jbertran

* @carlito-scality

* @slaperche-scality

* @nicolast

* @teddyandrieux

* @monpote

* @sayf-eddine-scality

* @nootal

* @alexandre-allard-scality

* @gdemonet

* @lucieleonard

* @thomasdanan

* @cmonfort





The following reviewers are expecting changes from the author, or must review again:

* @nicolast







*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/metalk8s/commit/341b19f5114ee91e1950ade706f2ba1a326e74e9) did not succeed in branch w/2.6/bugfix/add-kubeconfig-to-solutions-script.




*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.4`

* :heavy_check_mark: `development/2.5`

* :heavy_check_mark: `development/2.6`



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.4`

* :heavy_check_mark: `development/2.5`

* :heavy_check_mark: `development/2.6`



The following branches have **NOT** changed:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`



Please check the status of the associated issue None.

Goodbye ebaneck.
